integrations/angular/package.json

```diff
@@ -17,7 +17,7 @@
     "@angular/platform-browser": "~11.0.0",
     "@angular/platform-browser-dynamic": "~11.0.0",
     "@angular/router": "~11.0.0",
-    "airwallex-payment-elements": "*",
+    "airwallex-payment-elements": "0.2.11",
```

Comment:
We are propose for latest version, so I think don't need to specific the version

Reply:
Not sure why it didn't upgrade my package automatically on yarn when I tried it, but I will revert it to an asterisk.